CMakeLists.txt

```diff
@@ -63,6 +63,10 @@ option (Seastar_SSTRING
   "Use seastar's own string implementation"
   ON)
 
+option (Seastar_COLORED_LOG
+  "Print colored logging messages using ANSI escape code"
+  OFF)
```

Comment:
Can we make this auto-detected? Most people won't bother and will miss out.

Reply:
@tchaikov now it is auto-detected and enabled if fmt's version is greater or equal to 6.0.0 if this option is not specified explicitly.